wlauto/workloads/cameracapture/__init__.py

```diff
@@ -39,13 +39,20 @@ def __init__(self, device, **kwargs):
         super(Cameracapture, self).__init__(device, **kwargs)
         self.uiauto_params['no_of_captures'] = self.no_of_captures
         self.uiauto_params['time_between_captures'] = self.time_between_captures
+        self.uiauto_params['version'] = "button"
 
     def setup(self, context):
         super(Cameracapture, self).setup(context)
         self.device.execute('am start -n {}/{}'.format(self.package, self.activity))
 
+    def initialize(self, context):
+        if self.device.get_sdk_version() >= 23:
+            self.package = "com.google.android.GoogleCamera"
+            self.uiauto_params['version'] = "swipe"
```

Comment:
I would do this via a mapping -- that way if we have to update this in future android versions, we won't have a mass of if/elif statements.